common/dna_data/wc_human_dna.txt

```diff
@@ -0,0 +1,2870 @@
+			gene_bs_nose_ridge_angle={ "nose_ridge_angle_pos" 20 "nose_ridge_angle_pos" 30 }
+			gene_bs_nose_ridge_width={ "nose_ridge_width_pos" 22 "nose_ridge_width_pos" 22 }
+			gene_bs_nose_size={ "nose_size_pos" 28 "nose_size_neg" 5 }
+			gene_bs_nose_tip_angle={ "nose_tip_angle_pos" 39 "nose_tip_angle_neg" 28 }
+			gene_bs_nose_tip_forward={ "nose_tip_forward_pos" 47 "nose_tip_forward_neg" 23 }
+			gene_bs_nose_tip_width={ "nose_tip_width_pos" 21 "nose_tip_width_neg" 126 }
+			face_detail_cheek_def={ "cheek_def_02" 126 "cheek_def_02" 15 }
+			face_detail_cheek_fat={ "cheek_fat_01_pos" 5 "cheek_fat_01_pos" 5 }
+			face_detail_chin_cleft={ "chin_cleft" 36 "chin_cleft" 36 }
+			face_detail_chin_def={ "chin_def" 70 "chin_def" 70 }
+			face_detail_eye_lower_lid_def={ "eye_lower_lid_def" 253 "eye_lower_lid_def" 122 }
+			face_detail_eye_socket={ "eye_socket_02" 255 "eye_socket_01" 85 }
+			face_detail_nasolabial={ "nasolabial_01" 137 "nasolabial_01" 101 }
+			face_detail_nose_ridge_def={ "nose_ridge_def_pos" 120 "nose_ridge_def_pos" 4 }
+			face_detail_nose_tip_def={ "nose_tip_def" 153 "nose_tip_def" 24 }
+			face_detail_temple_def={ "temple_def" 211 "temple_def" 211 }
+			expression_brow_wrinkles={ "brow_wrinkles_01" 255 "brow_wrinkles_01" 40 }
+			expression_eye_wrinkles={ "eye_wrinkles_01" 255 "eye_wrinkles_01" 2 }
+			expression_forehead_wrinkles={ "forehead_wrinkles_02" 0 "forehead_wrinkles_02" 234 }
+			expression_other={ "nose_wrinkles_01" 255 "cheek_wrinkles_left_01" 127 }
+			complexion={ "complexion_3" 91 "complexion_7" 214 }
+			gene_height={ "normal_height" 128 "normal_height" 116 }
+			gene_bs_body_type={ "body_fat_head_fat_low" 137 "body_fat_head_fat_low" 134 }
+			gene_bs_body_shape={ "body_shape_average_clothed" 135 "body_shape_rectangle_full" 0 }
+			gene_bs_bust={ "bust_clothes" 84 "bust_shape_1_half" 84 }
+			gene_age={ "old_3" 0 "old_3" 205 }
+			gene_eyebrows_shape={ "avg_spacing_high_thickness" 167 "avg_spacing_low_thickness" 146 }
+			gene_eyebrows_fullness={ "layer_2_high_thickness" 138 "layer_2_lower_thickness" 133 }
+			gene_body_hair={ "body_hair_sparse_low_stubble" 255 "body_hair_avg" 134 }
+			hairstyles={ "all_hairstyles" 195 "western_hairstyles" 173 }
+			beards={ "all_beards" 24 "western_beards" 49 }
+			eye_accessory={ "normal_eyes" 231 "normal_eyes" 231 }
+			teeth_accessory={ "normal_teeth" 0 "normal_teeth" 0 }
+			eyelashes_accessory={ "normal_eyelashes" 176 "normal_eyelashes" 176 }
+			gene_bs_ear_lenght={ "floppy_ear_length" 0 "floppy_ear_length" 0 }
+			special_eyebrows={ "no_eyebrows" 14 "no_eyebrows" 14 }
+			special_eyes={ "no_special_eyes" 255 "no_special_eyes" 0 }
+			gene_bs_mouth_tusks={ "mouth_tusks_pos" 0 "mouth_tusks_pos" 0 }
+			static_tusks={ "no_static_tusks" 0 "no_static_tusks" 0 }
+			gene_bs_ear_tip={ "ear_tip_pos" 0 "ear_tip_pos" 0 }
+            gene_skin_value={ "skin_value_neg" 0 "skin_value_neg" 0 }
+		}
+	}
+	enabled=yes
+}
+
+nathanos_marris_undead_dna = { #by ercarp
```

Comment:
Are we using it?

Reply:
Because of the custom undead portrait modifier for Nathanos, no, I don't think so.

Reply:
So could you remove it?

Reply:
Removed in ed5cd8f2efd73481cb3dcf379af77f208a2ef5df.